scripts/zones/Grand_Palace_of_HuXzoi/mobs/Ixaern_MNK.lua

```diff
@@ -5,56 +5,98 @@
 local ID = require("scripts/zones/Grand_Palace_of_HuXzoi/IDs")
 require("scripts/settings/main")
 require("scripts/globals/status")
+require("scripts/globals/items")
 -----------------------------------
 local entity = {}
 
+local bracerMode = function(mob, qnAern1, qnAern2)
+    local mobID = mob:getID()
+    mob:useMobAbility(690) -- Hundred Fists
+    if qnAern1:isAlive() then qnAern1:useMobAbility(692) end -- Chainspell
+    if qnAern2:isAlive() then qnAern2:useMobAbility(689) end -- Benediction
+    mob:addMod(xi.mod.ATT, 200)
+
+    for i = mobID+1, mobID+2 do
+        local pet = GetMobByID(i)
+        if pet:isSpawned() then
+            pet:AnimationSub(2)
+            pet:addMod(xi.mod.ATT, 200)
+            pet:setMod(xi.mod.DELAY, 2100)
```

Comment:
that modifier does not work how you think it does. please stop using it. We need to do some reworking in regard to delay setting because the various ways to set/alter delays are inconsistent and causing confusion. `xi.mod.DELAY` is used for % reductions (see blitzers roll). You want the function `mob:setDelay(2100)`  instead (I think) for the input you are trying to give this.

A similar mixup happened on ladybugs and will need fixed.